### PR TITLE
Evaluation Administration List - adjust to make it mobile responsive …

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
[Evaluation Administration List - adjust to make it mobile responsive#441](https://github.com/nerubia/kh360-react/issues/441)

### CODE REVIEW

File: __tests__/pages/admin/evaluation-administrations/evaluation-administrations.spec.ts [hunk 1]
```diff
@@ -114,14 +114,30 @@ test.describe("Admin - Evaluation administrations", () => {
       await expect(page.getByRole("heading", { name: "Evaluations" })).toBeVisible()
       await expect(page.getByRole("link", { name: "Create Evaluations" })).toBeVisible()
 
-      await expect(page.getByRole("cell", { name: "Name" })).toBeVisible()
-      await expect(page.getByRole("cell", { name: "Period" })).toBeVisible()
-      await expect(page.getByRole("cell", { name: "Schedule" })).toBeVisible()
-      await expect(page.getByRole("cell", { name: "Status" })).toBeVisible()
+      if (!isMobile) {
```
Comment (jatnerubia) [inline __tests__/pages/admin/evaluation-administrations/evaluation-administrations.spec.ts:117]: use `if else` here

File: src/features/admin/evaluation-administrations/evaluation-administrations-table.tsx [hunk 2]
```diff
@@ -39,45 +39,89 @@ export const EvaluationAdministrationsTable = () => {
...
-        <thead className='text-left'>
-          <tr>
-            <th className='pb-3'>Name</th>
-            <th className='pb-3'>Period</th>
-            <th className='pb-3'>Schedule</th>
-            <th className='pb-3'>Status</th>
-          </tr>
-        </thead>
-        <tbody>
+      <span className='md:block hidden'>
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/evaluation-administrations-table.tsx:42]: this should be `div` 🤔

File: src/features/admin/evaluation-administrations/evaluation-administrations-table.tsx [hunk 3]
```diff
@@ -39,45 +39,89 @@ export const EvaluationAdministrationsTable = () => {
...
+                  >
+                    <div className='uppercase'>{evaluationAdministration?.status}</div>
+                  </Badge>
+                </td>
+              </tr>
+            ))}
+          </tbody>
+        </table>
+      </span>
+      <div className='w-full table-fixed md:hidden' data-testid='eval-list'>
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/evaluation-administrations-table.tsx:83]: do we need `table-fixed` here? 🤔

File: src/utils/format-date.ts [hunk 4]
```diff
@@ -4,7 +4,37 @@ import { type User } from "../types/user-type"
 export const formatDate = (date?: string) => {
   return date?.split("T")[0]
 }
+export const shortenFormatDate = (dateString?: string): string => {
```
Comment (jatnerubia) [inline src/utils/format-date.ts:7]: use `format` here? 🤔

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

File: src/utils/format-date.ts (hunk 4 above)
Comment (chadiegilnerubia) [reply src/utils/format-date.ts:7]: done :3

File: src/features/admin/evaluation-administrations/evaluation-administrations-table.tsx (hunk 3 above)
Comment (chadiegilnerubia) [reply src/features/admin/evaluation-administrations/evaluation-administrations-table.tsx:83]: done

File: src/features/admin/evaluation-administrations/evaluation-administrations-table.tsx (hunk 2 above)
Comment (chadiegilnerubia) [reply src/features/admin/evaluation-administrations/evaluation-administrations-table.tsx:42]: done

File: __tests__/pages/admin/evaluation-administrations/evaluation-administrations.spec.ts (hunk 1 above)
Comment (chadiegilnerubia) [reply __tests__/pages/admin/evaluation-administrations/evaluation-administrations.spec.ts:117]: done

**Review by emmaacha — APPROVED** [review]

**Review by jatnerubia — APPROVED** [review]
Nice

**Pull request merged**